### ISSUE TITLE
Possible to configure nginx subfolder?

### ISSUE THREAD

**Issue by ScottJWhite** [Hubro/clipface#44]
Is it possible to configure clipface on a subfolder? It works great if it place in the root location block in my nginx config, but if I try to do it on a subfolder I get 404 errors.

something like /clips instead of the root?:

location /clips {
        proxy_pass http://clipface/;
        proxy_set_header X-Forwarded-Host $host;
        proxy_set_header X-Forwarded-Proto $scheme;
        proxy_connect_timeout 3s;
        proxy_send_timeout 10s;
        proxy_read_timeout 300s;
        client_max_body_size 100m;
    }

**Comment by Hubro** [comment]
Good point, that's a really useful feature. It should also be pretty trivial to implement.

**Comment by atutr001** [comment]
I managed to forward it to a custom URL path using reverse proxy but it got confused once we get there cause all of the clips path is /watch
I guess there needs to be a parameter that converts every subsequent URL to be behind the existing custom URL. I might be talking nonsense anw, it's my first time using a reverse proxy lol. 
![image](https://user-images.githubusercontent.com/75003853/144759003-df019044-ac0e-434c-b20c-08eb7b8fbc0f.png)
![image](https://user-images.githubusercontent.com/75003853/144759015-ce188bea-b55f-440d-8440-48461ad1b010.png)
